---
get_next_case:
  replanning: false
  message: Put case on conveyor belt.
perceive_case_on_conveyor:
  replanning: false
  message: No Case detected on conveyor belt.
perceive_case_on_fixture:
  replanning: false
  message: No Case detected on fixture.
move_arm:
  replanning: false
  message: Could not move the arm.
pick_case:
  replanning: false
  message: Could not pick Case.
place_case:
  replanning: false
  message: Could not place Case in fixture.
perceive_insert:
  replanning: false
  message: Could not detect inserts.
pick_insert:
  replanning: false
  message: Could not pick Insert.
insert:
  replanning: false
  message: Not able to insert into Case.
inspect:
  replanning: false
  message: No result received from Set inspection.
perceive_set:
  replanning: false
  message: Could not detect Set.
pick_set:
  replanning: false
  message: Not able to pick Set.
place_set_in_box:
  replanning: false
  message: Could not place Set in box.
